### ISSUE TITLE
Startup errors with v2:  Entity id already exists & unsupported operand on round(gallons * 3.785, 2)

### ISSUE THREAD

**Issue by kkellner** [sebr/bhyve-home-assistant#23]
Home Assistant 0.111.4

Updated to newest version 2.0 of bhybe integration and now get the following errors at startup:

Error 1 
```
Logger: homeassistant.components.sensor
Source: custom_components/bhyve/sensor.py:163
Integration: Sensor (documentation, issues)
First occurred: 10:03:03 PM (6 occurrences)
Last logged: 10:03:03 PM

bhyve: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 319, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 470, in async_device_update
    await self.async_update()
  File "/config/custom_components/bhyve/sensor.py", line 163, in async_update
    litres = round(gallons * 3.785, 2)
TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'
```

Error 2
```
Logger: homeassistant.components.switch
Source: helpers/entity_platform.py:433
Integration: Switch (documentation, issues)
First occurred: 10:03:02 PM (5 occurrences)
Last logged: 10:03:02 PM

Entity id already exists - ignoring: switch.yukon_rain_delay. Platform bhyve does not generate unique IDs
Home Assistant has started!
```

**Comment by sebr** [comment]
@kkellner thanks for the report! I think the issues are fixed with 2.0.1.

**Comment by kkellner** [comment]
Confirmed that 2.0.1 fixed both issues.   Nice job on this integration!
